Resources/Textures/Effects/weapons/arcs.rsi/meta.json

```diff
@@ -4,7 +4,8 @@
         "x": 32,
         "y": 32
     },
-
+    "license": "CC-BY-SA-3.0 for bite->smash",
```

Comment:
This is a bit suspicious - the order of states is not guaranteed.

Reply:
time to split the files
tg_arcs.rsi

Reply:
I would support that - separates the copyright info nicely.

Reply:
FYI, `license` *should* be an SPDX license identifier so custom wording like this is a no-no.